```
github.com/docker/docker/vendor/github.com/moby/buildkit/solver/llbsolver/file
[2020-01-31T01:16:13.821Z] vendor/github.com/moby/buildkit/solver/llbsolver/file/user_nolinux.go:12:63: undefined: "github.com/docker/docker/vendor/github.com/tonistiigi/fsutil/copy".ChownOpt
script returned exit code 2
```

--- a/vendor.conf
+++ b/vendor.conf
@@ -26,8 +26,8 @@ github.com/imdario/mergo                            7c29201646fa3de8506f70121347
 golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
 
 # buildkit
-github.com/moby/buildkit                            926935b590c94c3659ebcc49cf44da47c1a65ff6
-github.com/tonistiigi/fsutil                        0f039a052ca1da01626278199624b62aed9b3729
+github.com/moby/buildkit                            57e8ad52170d713233569f6d467e609d2b0f90c9
+github.com/tonistiigi/fsutil                        6c909ab392c173a4264ae1bfcbc0450b9aac0c7d
 github.com/grpc-ecosystem/grpc-opentracing          8e809c8a86450a29b90dcc9efbf062d0fe6d9746
 github.com/opentracing/opentracing-go               1361b9cd60be79c4c3a7fa9841b3c132e40066a7
 github.com/google/shlex                             6f45313302b9c56850fc17f99e40caebce98c716

